app/src/main/java/net/bible/android/view/activity/bookmark/ManageLabelItemAdapter.kt

```diff
@@ -72,10 +72,6 @@ class ManageLabelItemAdapter(context: Context?,
                 if (data.showCheckboxes) {
                     checkbox.setOnCheckedChangeListener { _, isChecked ->
                         if (isChecked) {
-                            if (!manageLabels.selectMultiple) {
-                                data.selectedLabels.clear()
-                                data.bookmarkPrimaryLabel = null
-                            }
```

Comment:
why this change?

Reply:
I think this was a remnant of when you could choose to select either single or multiple labels. Didn't you hide that option and then no one complained about it so now we can only choose multiple labels. Thus that code is no longer needed.